pkg/client/linstor.go

```diff
@@ -636,6 +653,30 @@ func (s *Linstor) Detach(ctx context.Context, volId, node string) error {
 		"targetNode": node,
 	})
 
+	ress, err := s.client.Resources.GetAll(ctx, volId)
+	if err != nil {
```

Comment:
```suggestion
	if nil404(err) != nil {
```

Otherwise we fail the CSI tests because a "double" delete will result in an error

Reply:
Or actually just `return nil404(err)` in the branch